dask/dataframe/groupby.py

```diff
@@ -2308,7 +2310,10 @@ def aggregate(self, arg, split_every=None, split_out=1, shuffle=None):
     @_aggregate_docstring(based_on="pd.core.groupby.SeriesGroupBy.agg")
     def agg(self, arg, split_every=None, split_out=1, shuffle=None):
         return self.aggregate(
-            arg, split_every=split_every, split_out=split_out, shuffle=shuffle
+            arg,
+            split_every=split_every,
+            split_out=split_out,
+            **({} if shuffle is None else {"shuffle": shuffle}),
```

Comment:
IIUC this can go back to `shuffle=shuffle` once relevant updates have been made in `dask-cudf` -- if that's the case could we open an issue in `dask/dask` so we remember to follow-up and add a comment here linking to the issue?